app/logic/stake_logic.rb

```diff
@@ -152,8 +152,11 @@ def update_validator_stats
         uptimes = []
         lifetimes = []
         scores = []
+        total_active_stake = 0
 
-        Validator.where(id: validator_ids).each do |validator|
+        Validator.where(id: validator_ids).includes(:stake_accounts).each do |validator|
+          stake_acc = validator.stake_accounts.active.where(stake_pool: pool).first
```

Comment:
Please explain why we do we only refer to the first stake account? Instead of summing up the active stake from all stake accounts?